bin/update

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
This script should quit immediately if it fails.
So, should `set -ex`.

Reply:
I recommend `Makefile` to provide development tasks, which is `set -ex` by default for all the script execution.

BTY this is not the "update" task. It seems "release" or "publish".

Reply:
Added Makefile instead of `bin/update` at https://github.com/bitjourney/react-simple-image/pull/24/commits/7305e7be4db875046f91aae28aadfb2bac0b8208